tests/sites/version_switcher/_static/switcher.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
I think this isn't totally a self-contained test site: if the current `pydata_sphinx_theme.__version__` is `dev`, the first element will be selected in the version switcher; if it's `v0.16.1`, it'll be the second element. Not sure what happens if there's a different released version e.g. `v0.17.0`.

Why does this matter? The locator in the `test_version_switcher_highlighting` matches the text `dev`, specifically, so will either succeed or fail in locating the version switcher depending on the current `__version__`.

Reply:
This is a self-contained test. Since we are building the test site from `tests/version_switcher` and running the tests on this site, the version switcher will only ever display the versions in `tests/version_switcher/_static/switcher.json` so having a new release in PST would not affect this as we are not building based on the PST version itself.

I tested by changing our version in `__init__version.py` to an RC and a concrete version, and the tests and build of the test site worked as expected.

Reply:
Oh, right - it has its own `conf.py`, which is where our docs have that logic that defines the version matching for the version switcher :facepalm:. Sorry about this, and thanks for pointing this out!